tests/unit/Storefront/Controller/ContextGatewayControllerTest.php

```diff
@@ -146,7 +146,9 @@ public function testGatewayWithCustomerException(): void
 
     private function createStubContainerWithFlashBag(): ContainerInterface
     {
-        $session = new Session(new TestSessionStorage());
+        $storage = new TestSessionStorage();
+        $storage->clear(); // ensure a clean state since storage is a stub shared between tests
+        $session = new Session();
```

Comment:
```suggestion
        $session = new Session($storage);
```

or is that not necessary anymore? :thinking: